modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Inline toolbars flicker when switching between editors #TINY-8594 #TINY-8503
```

Comment:
This still needs fixing per https://github.com/tinymce/tinymce/pull/7748#discussion_r850085979

Reply:
I added a separated changedlog entry for TINY-8503